### ISSUE TITLE
NuGet Package Problem?

### ISSUE THREAD

**Issue by ozzy1873** [Windows-XAML/Template10#269]
I installed 1.0.6 and the title bar is now shifted. See attached pic.
![template10prob](https://cloud.githubusercontent.com/assets/8699636/10667127/eada2abe-7892-11e5-858b-e68a8e24156b.png)

**Comment by ozzy1873** [comment]
If I resize the window to it's smallest width, the space closes, but there is still no margin between the text and the left edge of the title bar.

**Comment by JerryNixon** [comment]
You are the second to ask this question. The answer is that you need to ensure the PageHeader has the VisualStateNarrowMinWidth set to the same value you set your HamburgerMenu. But before you do that, could you help me understand how this is happening. Is this a vanilla Minimal project? How did this property get out of sync?

**Comment by ozzy1873** [comment]
My project is based on the Search sample. I was using NuGet package 1.0.4 and the title bar was behaving normally. When package 1.0.6 was released, I simply updated to the new package with no other code changes. This is when the problem started. I did not explicitly set the VisualStateNarrowMinWidth property for either the PageHeader or the HamburgerMenu. If I revert to 1.0.4, the title bar once again behaves normally.
